### ISSUE TITLE
FAP reviewers should not be able to download HPL tech spec in proposal summary

### ISSUE THREAD

**Issue by simonfernandes** [UserOfficeProject/user-office-project-issue-tracker#1019]
Links with https://github.com/UserOfficeProject/issue-tracker/issues/956

When HPL use the FAP functionality, the HPL tech spec should be hidden from FAP reviewers:

- When they review the question and answers on the proposal overview page, they should not be able to download the individual tech spec attachment.
![image](https://github.com/UserOfficeProject/issue-tracker/assets/14293717/edf18644-e5f0-447a-92e2-833fdcd13627)

- When they download the proposal PDF, the tech spec should not be included. (Covered by https://github.com/UserOfficeProject/issue-tracker/issues/956)

**Comment by joshhdawes** [comment]
@Cat-Lucifer this has been on a list of things to look into for a while - is it a priority?

**Comment by Cat-Lucifer** [comment]
@joshhdawes  This is one to consider as part of the meeting work, when we come to do that for CLF.  So, not a priority now, but will be then (whenever that will be - mid-second half of next year sometime I expect).  In the meantime, I can easily do what's needed manually for the current process.

**Comment by joshhdawes** [comment]
From reqs meeting on 30/5/25 - @Cat-Lucifer is going to ask HPL whether this really needs to be done for the coming round. We'll decide whether or not to do it based on the response.

**Comment by Cat-Lucifer** [comment]
@joshhdawes @TCMeldrum I have checked with HPL, and this is a priority for them and it needs implementing.  There are two aspects to this:

1. The Tech Spec upload doc must not being viewable in the proposal pdf for FAP Chairs or FAP Reviewers, but must be viewable in the pdf for all other roles (User, Experiment Scientist, Internal Reviewer, User Officer, FAP Secretary)
2. The Tech Spec upload question must not be viewable in the Proposal Info table in UOP for FAP Chairs or FAP Reviewers, but must viewable for all other roles

I think HPL also want more of the proposal questionnaire questions hidden for FAP Chairs or FAP Reviewers in the Proposal Info table.  The comment given around this is:

"The only additional information to the proposal pdf the FAP reviewers should see is the facility technical assessment."

HPL are keen to test the system changes themselves prior to final roll out.

Good luck!

**Comment by mehta-pooja123** [comment]
@joshhdawes  @TCMeldrum @Cat-Lucifer : Slightly off topic but I am considering this issue as a requirement for our roles and permissions discussion.

**Comment by TCMeldrum** [comment]
Yeah sure, not sure on the timeline of the roles work but we might have to do something beforehand.

**Comment by mehta-pooja123** [comment]
[like]  Mehta, Pooja (STFC,RAL,ISIS) reacted to your message:
________________________________

**Comment by TCMeldrum** [comment]
I think think the easiest way to do this would be to tag the questions as internal/external. Then we have some mapping between the current hard coded roles and if they can see internal questions. When it comes time to do the roles work we would then be able to tag the dynamic roles are internal/external as well. 

How does that sound for other facilities? @UserOfficeProject/swap @gnyiri
